.pipelines/templates/template-prod-e2e-steps.yml

```diff
@@ -21,6 +22,10 @@ steps:
   parameters:
     workingDirectory: ${{ parameters.workingDirectory }}
     azureDevOpsJSONSPN: ${{ parameters.azureDevOpsE2EJSONSPN }}
+- template: ./templates/template-push-images-to-acr.yml
+  parameters:
+    workingDirectory: ${{ variables.modulePath }}
+    rpImageACR: ${{ parameters.rpImageACR }}
```

Comment:
I don't think this is right.  It will be necessary to build and push an image as part of the CI flow, but it should not be done in the int or prod deployment flows.